app/web/features/profile/view/userLabels.tsx

```diff
@@ -180,13 +180,20 @@ const AgeAndGenderRenderer = ({ user }: Props) => {
     }
   };
   return (
-    <>
+    <div
+      style={{
+        display: "flex",
+        flexWrap: "wrap",
+        alignItems: "center",
+      }}
+    >
       <span>{age}</span>
-      {getBirthdateVerificationIcon(birthdateVerificationStatus)} /&nbsp;
+      {getBirthdateVerificationIcon(birthdateVerificationStatus)}
+      <span>/&nbsp;</span>
```

Comment:
Is the goal to add a space after the icon? If so please do it via css.

Reply:
The goal was to put a space after the number/symbol because I saw inconsistent styling. 
eg. It looked like this before: "28 /Man" and it looks like this with the change: "28 / Man"